docs/wcs/note_sip.rst

```diff
@@ -0,0 +1,40 @@
+.. include:: references.rst
+.. doctest-skip-all
+.. _note_sip:
+
+Note about SIP and WCS
+======================
+
+
+`astropy.wcs`, as well as `wcslib`_, include the `SIP`_ distortion
+in the WCS object if SIP coefficients are found in the header.
+The presence of a "-SIP" suffix in CTYPE is not used as a trigger
+to initialize the SIP distortion.
```

Comment:
It might be worth first explaining before this paragraph that the strictly correct way to implement SIP is that the header needs the SIP coefficients **and** the CTYPEs need the ``-SIP`` suffix. Then you can say that however since this has not been a strict convention in the past, the default in Astropy is to always interpret the SIP coefficients if present even if ``-SIP`` is not present.